packages/libdovi.cmake

```diff
@@ -0,0 +1,54 @@
+if(${TARGET_CPU} MATCHES "x86_64")
+    set(libdovi_target "x86_64-pc-windows-gnu")
+else()
+    set(libdovi_target "i686-pc-windows-gnu")
+endif()
+
+ExternalProject_Add(libdovi
+    GIT_REPOSITORY https://github.com/quietvoid/dovi_tool.git
+    SOURCE_DIR ${SOURCE_LOCATION}
+    GIT_CLONE_FLAGS "--filter=tree:0"
+    GIT_TAG libdovi-3.1.1
+    UPDATE_COMMAND ""
+    PATCH_COMMAND ""
+    CONFIGURE_COMMAND ""
+    BUILD_COMMAND ""
+    INSTALL_COMMAND ""
+    LOG_DOWNLOAD 1 LOG_UPDATE 1
+)
+
+ExternalProject_Add_Step(libdovi setup-toolchain
+    DEPENDEES install
+
+    COMMAND ${EXEC} rustup install stable
+    COMMAND ${EXEC} rustup target add ${libdovi_target}
+    COMMAND ${EXEC} rustup update
+    COMMAND ${EXEC} rustup default stable
```

Comment:
I think this step isn't necessary anymore, as the `rustup` toolchain package already sets up stable + updating.

Reply:
sure